core/trino-main/src/main/java/io/trino/operator/partition/PagePartitioner.java

```diff
@@ -0,0 +1,264 @@
+/*
```

Comment:
`Move PagePartitioner to separate file` -> `Extract PagePartitioner as a top-level class`